CRM/Upgrade/Incremental/sql/5.56.alpha1.mysql.tpl

```diff
@@ -14,3 +14,9 @@ SET ca.state_province_id = @DHStateID
 WHERE csp.country_id = @indianCountryID AND csp.abbreviation IN ("DN", "DD");
 
 UPDATE civicrm_state_province SET is_active = 0 WHERE country_id = @indianCountryID AND abbreviation IN ("DN", "DD");
+
+-- Fix incorrect civicrm_preferences_date description for activityDate and searchDate
+
+UPDATE civicrm_preferences_date SET description = {localize}'{ts escape="sql"}Date for relationships. activities. contributions: receive, receipt, cancel. membership: join, start, renew. case: start, end.{/ts}'{/localize} WHERE civicrm_preferences_date.name = 'activityDate';
```

Comment:
@agileware-justin just remove `{localize}` and `{/localize}` here and you should be fine and in L22

Reply:
Trial and error, waiting for the build to confirm.